docs/OpenBSD.md

```diff
@@ -48,23 +48,14 @@ Install OpenBSD over PXE
   	menu default
   	kernel OpenBSD/amd64/pxeboot.0
```

Comment:
@pkubaj please consider adding some earlyprint over COM. Running this option in PXE menu causes no output and restarts apu2 after a while